src/middlewared/middlewared/plugins/iscsi.py

```diff
@@ -1008,14 +1008,51 @@ class Config:
         # if not data['groups']:
         #    verrors.add(f'{schema_name}.groups', 'At least one group is required')
 
+        db_portals = list(
+            map(
+                lambda v: v['id'],
+                await self.middleware.call('datastore.query', 'services.iSCSITargetPortal', [
+                    [
+                        'OR', [
+                            ('id', '=', portal_id) for portal_id in map(lambda v: v['portal'], data['groups'])
+                        ]
```

Comment:
Isnt OR somewhat overkill? Wouldnt ('id', 'in', portal_ids) do just the same?

Reply:
It would, thank you :)